### PR TITLE
LinkedIn - How to connect rule - Integrating Adam's feedback

### PR DESCRIPTION
**Pull request by Freego1783**
>
> 1. What triggered this change? (PBI link, Email Subject, conversation + reason, etc)

Adam's feedback on the rule

> 2. What was changed?

Added more people in authors section
Linked the rule about weekly client love
Changed the way some sentences were written
Added a bad example

> 3. Did you do pair or mob programming (list names)?

No

### CODE REVIEW

File: rules/linkedin-connect-with-people/rules.md [hunk 1]
```diff
@@ -6,30 +6,52 @@ guid: 12a2e1c9-fc4e-4c10-9c4f-f11ed665e668
...
 <!--endintro-->
 
-1. On the LinkedIn mobile app, avoid using the default "Connect" or "Message" buttons. Instead, use the "ellipsis | Personalize invite" button to craft a custom message. On the LinkedIn website, click "Connect | Add a note"
+1. On the LinkedIn mobile app - avoid using the default "Connect" or "Message" buttons. Instead, use the "ellipsis | Personalize invite" button to craft a custom message.
 
+   On the LinkedIn website - click "Connect | Add a note"
+
+::: good
 ![Figure: Use "Personalize invite" instead of "Connect" or "Message"](personnalize-invite.jpeg)
+:::
```
Comment (bradystroud) [inline rules/linkedin-connect-with-people/rules.md:32]: I think you also need a bad example here

**Review by bradystroud — CHANGES_REQUESTED** [review]

File: rules/linkedin-connect-with-people/rules.md [hunk 2]
```diff
@@ -6,30 +6,52 @@ guid: 12a2e1c9-fc4e-4c10-9c4f-f11ed665e668
...
 
-By using these wisely, you can create personalized and memorable connection experiences, making it easier to build and maintain professional relationships.
+By using this technique, you can create memorable connection experiences, making it easier to build and maintain professional relationships down the track.
 
 ::: info
 **Monthly limit**
 
 By default you only have 5 custom messages per month, which should be enough. If you want to be able to send more customized messages you will need to upgrade your LinkedIn plan to Premium.
+
+More information there: [Personalize invitations to connect](https://www.linkedin.com/help/linkedin/answer/a563153)
```
Comment (tkapa) [inline rules/linkedin-connect-with-people/rules.md:56]:
```suggestion
More information: [Personalize invitations to connect](https://www.linkedin.com/help/linkedin/answer/a563153)
```

File: rules/linkedin-connect-with-people/rules.md [hunk 3]
```diff
@@ -6,30 +6,52 @@ guid: 12a2e1c9-fc4e-4c10-9c4f-f11ed665e668
...
+    url: https://ssw.com.au/people/adam-cogan/
+  - title: Brady Stroud
+    url: https://ssw.com.au/people/brady-stroud/
+  - title: Tylah Kapa
+    url: https://ssw.com.au/people/tylah-kapa/
   - title: Hugo Pernet
     url: https://ssw.com.au/people/hugo-pernet/
 related:
   - successful-reachouts
+  - weekly-client-love
```
[the remark this answers is not in the document]
Comment (Freego1783) [reply rules/linkedin-connect-with-people/rules.md:19]: I tought I would add it after the other rule is fixed because now it's not really related

**Review by tkapa — APPROVED** [review]
Approved with Changes

File: rules/linkedin-connect-with-people/rules.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Freego1783) [reply rules/linkedin-connect-with-people/rules.md:19]: ✅ Done

File: rules/linkedin-connect-with-people/rules.md (hunk 1 above)
Comment (Freego1783) [reply rules/linkedin-connect-with-people/rules.md:33]: Don't know what to add there, the screenshot actually shows what's bad (clicking connect directly) and what's good (using personalize invite)

**Review by bradystroud — APPROVED** [review]

**Pull request merged**